pkg/cstorpool/v1alpha2/cstorpool_test.go

```diff
@@ -0,0 +1,167 @@
+package v1alpha2
+
+import (
+	"testing"
+
+	apis "github.com/openebs/maya/pkg/apis/openebs.io/v1alpha1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+)
+
+func TestAll(t *testing.T) {
+	tests := map[string]struct {
+		mockPredicates predicateList
+		expectedOutput bool
+	}{
+		// Positive predicates
+		"Positive Predicate 1": {[]predicate{func(*csp) bool { return true }}, true},
```

Comment:
redundant code can be written as functions e.g. `fakeAlwaysTrue`, `fakeAlwaysFalse` can be two functions that satisfy the predicate type.

Reply:
done